x/wasm/keeper/keeper.go

```diff
@@ -526,15 +558,59 @@ func (k Keeper) migrate(
 		sdk.NewAttribute(types.AttributeKeyContractAddr, contractAddress.String()),
 	))
 
-	sdkCtx = types.WithSubMsgAuthzPolicy(sdkCtx, authZ.SubMessageAuthorizationPolicy(types.AuthZActionMigrateContract))
-	data, err := k.handleContractResponse(sdkCtx, contractAddress, contractInfo.IBCPortID, res.Ok.Messages, res.Ok.Attributes, res.Ok.Data, res.Ok.Events)
-	if err != nil {
-		return nil, errorsmod.Wrap(err, "dispatch")
+	var data []byte
+	if response != nil {
```

Comment:
```suggestion

	// if migrate entry point was called
	if response != nil {
```

Is this comment correct?

Reply:
Yes, the comment is correct. response is only set if we called the migrate entrypoint above.